lib/NormalModule.js

```diff
@@ -178,6 +182,19 @@ class NormalModule extends Module {
 		return loaderContext;
 	}
 
+	purifyCurrentLoaderName(loaderContext) {
+		const currentLoaderObject = this.loaders && this.loaders.length ?
+			this.loaders[loaderContext.loaderIndex] :
+			null;
+		if(currentLoaderObject) {
+			const loaderName = currentLoaderObject.origin || currentLoaderObject.loader;
+			return /\/|\\/.test(loaderName) && path.isAbsolute(loaderName) ?
```

Comment:
We have a `requestShortener` for this. `compilation.runtimeTemplate.requestShortener`